src/pages/StoryForm/StoryForm2.css

```
@@ -0,0 +1,256 @@
+.story-form2-container {
+  padding-bottom: 40px;
+}
+
+/* 사연 본문 */
+
+.story-form2-textarea-wrapper {
+  position: relative;
+
+  width: 100%;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)